pandas/tests/dtypes/test_inference.py

```diff
@@ -794,6 +794,12 @@ def test_infer_dtype_datetime(self):
             arr = np.array([n, datetime(2011, 1, 1)])
             assert lib.infer_dtype(arr, skipna=True) == "datetime"
 
+            arr = np.array([n, datetime(2011, 1, 1, tzinfo=pytz.utc)])
```

Comment:
can you move these to a new test

Reply:
Done. I took the liberty of adding some more tests for cases that seem relevant to me. I hope that's alright.